src/screens/CreateToken.js

```diff
@@ -249,6 +196,7 @@ class CreateToken extends React.Component {
         <p className="mt-5">Here you will create a new customized token. After the creation, you will be able to send this new token to other addresses.</p>
         <p>Custom tokens share the address space with all other tokens, including HTR. This means that you can send and receive tokens using any valid address.</p>
         <p>Remember to make a backup of your new token's configuration string. You will need to send it to other people to allow them to use your new token.</p>
+        <p>When creating and minting tokens, a <strong>deposit of {hathorLib.tokens.depositPercentage * 100}%</strong> in HTR is required. This will be returned when these tokens are later melted.</p>
```

Comment:
I would say: "... These will be returned if these tokens are melted later. For further information, see [link]".
I'm not sure about what link to use. Should we point to the RFC? Maybe we should have a document in our website.

Reply:
"When creating and minting tokens, a deposit of 1% in HTR is required. If these tokens are later melted, this HTR deposit will be returned. Read more about it here."

Linked to the RFC as we have no other document yet.